pkg/apis/messaging/v1beta1/kafka_channel_lifecycle.go

```diff
@@ -53,22 +79,57 @@ const (
+	//
 
-	// KafkaChannelConditionServiceReady has status True when a k8s Service representing the channel is ready.
-	// Because this uses ExternalName, there are no endpoints to check.
-	KafkaChannelConditionChannelServiceReady apis.ConditionType = "ChannelServiceReady"
+	// KafkaChannelConditionReceiverServiceReady has status True when the Receiver's K8S Service is ready.
+	// This basically just means it exists because there's no meaningful status in Service.
+	KafkaChannelConditionReceiverServiceReady apis.ConditionType = "ReceiverServiceReady"
 
-	// KafkaChannelConditionTopicReady has status True when the Kafka topic to use by the channel exists.
-	KafkaChannelConditionTopicReady apis.ConditionType = "TopicReady"
+	// KafkaChannelConditionReceiverDeploymentReady has status True when the Receiver's K8S Deployment is ready.
+	KafkaChannelConditionReceiverDeploymentReady apis.ConditionType = "ReceiverDeploymentReady"
 
-	// KafkaChannelConditionConfigReady has status True when the Kafka configuration to use by the channel exists and is valid
-	// (ie. the connection has been established).
-	KafkaChannelConditionConfigReady apis.ConditionType = "ConfigurationReady"
+	// KafkaChannelConditionDispatcherServiceReady has status True when the Dispatcher's K8S Service is ready.
+	// This basically just means it exists because there's no meaningful status in Service.
+	// Note - Dispatcher Service is only exposed for Metrics collection and is not an active part of the data-plane.
+	KafkaChannelConditionDispatcherServiceReady apis.ConditionType = "DispatcherServiceReady"
+
+	// KafkaChannelConditionDispatcherDeploymentReady has status True when the Receiver's K8S Deployment is ready.
+	KafkaChannelConditionDispatcherDeploymentReady apis.ConditionType = "DispatcherDeploymentReady"
 )
 
+// RegisterConsolidatedKafkaChannelConditionSet initializes the ConditionSet to those pertaining to the consolidated KafkaChannel.
+func RegisterConsolidatedKafkaChannelConditionSet() {
+	RegisterAlternateKafkaChannelConditionSet(
+		apis.NewLivingConditionSet(
+			KafkaChannelConditionAddressable,
+			KafkaChannelConditionConfigReady,
+			KafkaChannelConditionTopicReady,
+			KafkaChannelConditionChannelServiceReady,
+			KafkaChannelConditionDispatcherReady,
+			KafkaChannelConditionServiceReady,
+			KafkaChannelConditionEndpointsReady,
+		),
+	)
+}
+
+// RegisterDistributedKafkaChannelConditionSet initializes the ConditionSet to those pertaining to the distributed KafkaChannel.
+func RegisterDistributedKafkaChannelConditionSet() {
+	RegisterAlternateKafkaChannelConditionSet(
+		apis.NewLivingConditionSet(
+			KafkaChannelConditionAddressable,
+			KafkaChannelConditionConfigReady,
+			KafkaChannelConditionTopicReady,
+			KafkaChannelConditionChannelServiceReady,
+			KafkaChannelConditionReceiverServiceReady,
+			KafkaChannelConditionReceiverDeploymentReady,
+			KafkaChannelConditionDispatcherServiceReady,
+			KafkaChannelConditionDispatcherDeploymentReady,
+		),
+	)
+}
```

Comment:
Same comment for these.

Reply:
👍 will move everything channel specific together based on comment above!